libqtile/backend/base.py

```diff
@@ -253,6 +256,57 @@ def info(self) -> dict[str, Any]:
         """
         return {}
 
+    @expose_command()
+    def keep_above(self) -> None:
+        """Keep this window above all others"""
+
+    @expose_command()
+    def keep_below(self) -> None:
+        """Keep this window below all others"""
+
```

Comment:
I saw that `group.py` checks `win.kept_above`, requiring the property. Perhaps this would work instead?
```suggestion
    @expose_command()
    def keep_above(self, enable: bool | None = None) -> None:
        """Keep this window above all others"""

    @expose_command()
    def keep_below(self, enable: bool | None = None) -> None:
        """Keep this window below all others"""
```
By default, `enable = None` can mean 'toggle', otherwise it's specifying whether to keep above/below or not.